ambry-clustermap/src/test/java/com/github/ambry/clustermap/HelixParticipantTest.java

```diff
@@ -397,6 +398,7 @@ public void testGetAndSetReplicaStoppedState() throws Exception {
   public void testBadCases() {
     // Invalid state model def
     props.setProperty("clustermap.state.model.definition", "InvalidStateModelDef");
+    props.setProperty(ClusterMapConfig.ENABLE_PROPERTY_STORE_CLEAN_UP_TASK, Boolean.toString(false));
```

Comment:
I need to disable propertyStoreTask in this test since this is using a very specific Mock of HelixManager for testing for participate method instead of ZKHelixManager.